src/pages/ItemList.js

```diff
@@ -19,7 +19,7 @@ const ONE_DAY = ONE_HOUR * 24;
 function ItemList({ token }) {
   const [docs, setDocs] = useState([]);
   const [searchInput, setSearchInput] = useState('');
-  const [now, setNow] = useState(Date.now());
+  const [now] = useState(Date.now());
```

Comment:
Rather than holding `now` as state we could either make it a variable 
`const now = Date.now();`

or just invoke `Date.now()` in places where we are using `now` currently.

The reason for this is that `now` is actually just `Date.now()` when the component first renders and is then never updated. So, it's now actually "now" at the time of invocation, but rather referring to a time in the past (when the component first rendered).

Reply:
Thanks! Will do!